### PR TITLE
Open LinkedIn links in new tab and update to Marty

### PR DESCRIPTION
**Pull request by martycochrane**
- [x] Changed LinkedIn links on profiles to open in a new tab instead of redirecting from website
- [x] Added Marty's CTO role to summary

### CODE REVIEW

File: layouts/about/single.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1 class="center">{{ .Title }}</h1>
   <li><a href="mailto:{{ . }}">{{ . }}</a>
   {{- end }}
   {{ with .Params.Linkedin -}}
-  <li><a href="{{ . }}"><img src="/img/LI-In-Bug.png"></a>
+  <li><a href="{{ . }}" target="_blank"><img src="/img/LI-In-Bug.png"></a>
```
Comment (mandus) [inline layouts/about/single.html:21]: I disagree on this change - I honestly think it should be up to the viewer whether he wants a new window/tab or open in the same one.

**Review by mandus — COMMENTED** [review]
The change in personal entry is okay; I disagree on the other one...

File: layouts/about/single.html (hunk 1 above)
Comment (martycochrane) [reply layouts/about/single.html:21]: @mandus yeah that's a good point.
I'm just used to it being standard practise that if you're providing an external link then you don't redirect from your own website but I think it's fine either way. I'll revert the change now

**Review by mandus — APPROVED** [review]
lgtm

**Pull request merged**